components/director/internal/domain/application/service.go

```diff
@@ -52,6 +52,7 @@ type repoCreatorFunc func(ctx context.Context, tenant string, application *model
 type repoUpserterFunc func(ctx context.Context, tenant string, application *model.Application) (string, error)
 
 // ApplicationRepository missing godoc
+//
```

Comment:
Only formatting changes. I notice that for some files I also get different formatting lately, even without meaningful changes in the same file. Live it like this.